extensions/wikia/ArticleVideo/bower_components/html5-skin/js/views/wikiaAdScreen.js

```diff
@@ -0,0 +1,68 @@
+/********************************************************************
+ WIKIA AD SCREEN
+ *********************************************************************/
+var React = require('react'),
+  CONSTANTS = require('../constants/constants'),
+  ControlBar = require('../components/controlBar');
+
+var WikiaAdScreen = React.createClass({
+
+  getInitialState: function () {
+    return {
+      isMobile: this.props.controller.state.isMobile
+    };
+  },
+
+  getPlaybackControlItems: function () {
+    if (!this.props.skinConfig.adScreen.showControlBar) return null;
+
+    var showControlBar =
+      this.props.controller.state.controlBarVisible || this.props.playerState == CONSTANTS.STATE.PAUSE;
+
+    var playbackControlItemTemplates = {
+      "controlBar": <ControlBar {...this.props}
+                                controlBarVisible={showControlBar}
+                                playerState={this.props.playerState}
+                                isWikiaAdScreen={true}
+                                showAdTimeLeft={this.props.controller.state.showAdTimeLeft}
+                                showAdFullScreenToggle={this.props.controller.state.showAdFullScreenToggleToggle}
+                                key='controlBar'/>
+    };
+
+    var playbackControlItems = [];
+    for (var item in playbackControlItemTemplates) {
+      if (playbackControlItemTemplates.hasOwnProperty(item)) {
+        playbackControlItems.push(playbackControlItemTemplates[item]);
+      }
+    }
+
+    return playbackControlItems;
+  },
+
+  // FIXME move this to controller.js (?) so the first click shows controls, not toggles play/pause
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1134)